ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -152,3 +152,35 @@ def test_numpy_normal(
         scale=scale,
         size=size,
     )
+
+
+# shuffle
+@handle_frontend_test(
+    fn_tree="numpy.random.shuffle",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")
+    ),
+)
+def test_numpy_shuffle(
+        dtype_and_x,
+        as_variable,
+        num_positional_args,
+        native_array,
+        frontend,
+        fn_tree,
+        on_device,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
```

Comment:
We had a small refactor, you have to refactor the tests according to this.
https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py#L29
Thanks!

Reply:
Dear @zaeemansari70, Sorry for the inconvenience I caused you?, Can I close this PR, and open another PR? This one has gotten a little bit complicated since I worked on the master branch and I want to open a new PR by creating a branch.

Reply:
Sure, I'll close this PR as discussed. 🙂